packages/fxa-auth-server/docs/swagger/subscriptions-docs.ts

```diff
@@ -0,0 +1,215 @@
+
+      Validate and store a Play Store Puchase Token for the given user. Returns token validity.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_PLANS_GET = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Returns a list of available subscription plans.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_ACTIVE_GET = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Returns a list of active subscriptions for the user.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_CUSTOMER_POST = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Create a new customer object for use with subscription payments.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_ACTIVE_NEW_POST = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Subscribe the user to a price using a payment method id.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_INVOICE_RETRY_POST = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Retry an incomplete subscription invoice with a new payment method id.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_SETUPINTENT_CREATE_POST = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Create a new setup intent for attaching a new payment method to the user.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_PAYMENTMETHOD_DEFAULT_POST = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Update a user's default payment method for invoices to the attached payment method id.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_ACTIVE_SUBSCRIPTIONID_DELETE = {
+  description: '🔒 oauthToken',
+  notes: [
+    dedent`
+      🔒 authenticated with OAuth bearer token
+
+      Cancel an active subscription for the user.
+    `,
+  ],
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_PAYPAL_CHECKOUT_POST = {
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_MOZILLA_SUBSCRIPTIONS_CUSTOMER_BILLING_AND_SUBSCRIPTIONS_GET = {
+  description: '🔒 oauthToken',
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_IAP_RTDN_POST = {
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_CLIENTS_GET = {
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_INVOICE_PREVIEW_POST = {
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_INVOICE_PREVIEW_SUBSEQUENT_GET = {
+  tags: TAGS.SUBSCRIPTIONS,
+};
+
+const OAUTH_SUBSCRIPTIONS_COUPON_POST = {
+  tags: TAGS.SUBSCRIPTIONS,
+};
```

Comment:
Personally I'd create a single const:
```
const TAGS_SUBSCRIPTIONS = {
  tags: TAGS.SUBSCRIPTIONS,
}
```
and use that where you need it instead of separate constants that are all exactly the same value. You can then spread it where you need it:

```
const OAUTH_MOZILLA_SUBSCRIPTIONS_CUSTOMER_BILLING_AND_SUBSCRIPTIONS_GET = {
  description: '🔒 oauthToken',
  {...TAGS_SUBSCRIPTIONS}
};
```
Same comment applies to other files doing this.

Reply:
I thought about this and felt it would took up the same amount of space... 🤔?